Verifier.ts

```diff
@@ -36,43 +39,56 @@ export class Verifier extends Actor<Verifier> {
 						result =
 							splitted.length == 3 &&
 							algorithm &&
-							(await algorithm.verify(`${splitted[0]}.${splitted[1]}`, splitted[2]))
+							(await algorithm.some(async a => await a.verify(`${splitted[0]}.${splitted[1]}`, splitted[2])))
 								? result
 								: undefined
 					}
 				} catch {
 					result = undefined
 				}
-				const now = Date.now()
+				if (result) {
+					//Legacy problem, transform epoch in ms to epoch in s
+				}
+				result = result && this.verifyAudience(result.aud, audience) ? result : undefined
+			}
+			if (result)
+				result = await this.transformers.reduceRight(async (p, c) => c.reverse(await p), Promise.resolve(result))
+			if (result) {
+				// TODO make Pretty maybe find a new place
+				const now = Math.floor(Date.now() / 1000)
```

Comment:
Create an issue instead, we should not make an habit of using the code as a notebook.